packages/drawer/src/index.tsx

```diff
@@ -55,13 +82,105 @@ export class Drawer extends MaterialComponent<IDrawerProps, IDrawerState> {
     } else if (props.dismissible) {
       classes.push('mdc-drawer--dismissible');
     }
+    // I dont know what will be the type of this.
+    const styleObject: any = {};
+    if (this.state.sliderState !== SLIDER_STATES.REST) {
+      styleObject.display = 'flex';
+      styleObject.transform = `translateX(${this.state.swipeDistance}px)`;
+    }
+
+    if (this.state.sliderState === SLIDER_STATES.SWIPE_TO_BE_COMPLETED) {
+      styleObject['transition-duration'] = '300ms';
+    }
+
     return (
       <div>
-        <aside class={classes.join(' ')} ref={this.setControlRef} {...props}>
+        <aside
+          class={classes.join(' ')}
+          ref={this.setControlRef}
+          {...props}
+          style={styleObject}>
           {props.children}
         </aside>
         {props.modal && <div class="mdc-drawer-scrim" />}
       </div>
     );
   }
+
+  private handleTouchStart_ = e => {
```

Comment:
Why do they all have an underscore at the end?

Reply:
cuz they are private and we ship js not TS. WDYT?

Reply:
Most IDEs use the .d.ts file also to suggest things in JavaScript (at least WebStorm and VSCode I think)

Reply:
i know but 
a. lets not take that for granted
b. does it hurt? even MDC-web ships this way

Reply:
It's not consistent with the rest of the codebase and I think it's quite ugly, but if you really want to do this

Reply:
i'll remove it